bitnami/dataplatform-bp1/values.yaml

```diff
@@ -161,6 +228,30 @@ solr:
     requests:
       cpu: 250m
       memory: 5120Mi
+  ## Configuration for the solr prometheus exporter
+  ##
+  exporter:
+    enabled: '{{ .Values.wavefront.enabled }}'
+    ## Solr exporter port
+    ##
+    port: 9983
+    affinity:
+      podAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+         - weight: 50
+           podAffinityTerm:
+             labelSelector:
+               matchExpressions:
+               - key: app.kubernetes.io/name
+                 operator: In
+                 values:
+                   - solr
+             topologyKey: "kubernetes.io/hostname"
```

Comment:
Why are we adding these options? Not sure if this is something that should be enabled by default. @javsalgar what do you think?

Reply:
@marcosbc This is basically a solution chart to deploy kafka/spark/solr together. The intention is to provide the pod affinity rules + resource optimization to provide recommended kubernetes node count.

Reply:
AFAIK this is how the OCTO team wants the Helm Chart to be deployed by default. You can also see other affinity rules set in the rest of components

Reply:
yes thats right @miguelaeh

Reply:
Fine then